flowcraft/generator/templates/bandage.nf

```diff
@@ -0,0 +1,48 @@
+if (params.reference{{param_id}} != null) {
+    process bandage_{{pid}} {
+        {% include "post.txt" ignore missing %}
+
+        tag { sample_id }
+        publishDir "reports/assembly/bandage_{{pid}}/$sample_id"
+
+        input:
+        set sample_id, file(assembly) from {{input_channel}}
+        file reference from Channel.fromPath(params.reference{{param_id}})
+
+        output:
+        file "*"
```

Comment:
Could you be more specific here on the regex pattern? This will also include dot file (.command.*) from nextflow in the publishDir.